test/integration/cloud-config-aws.yml.template

```diff
@@ -14,6 +14,8 @@
 aws_access_key: @ACCESS_KEY
 aws_secret_key: @SECRET_KEY
 security_token: @SECURITY_TOKEN
+aws_region: @REGION
```

Comment:
Correcting myself a bit:
None of the legacy tests using AMIs have been ported to ansible-test yet. Merging this will make those tests need an alternative to hard coded AMIs before they can be run with CI. It would probably be good to have a solution in mind before merging.

We do have tests with hard coded AZ though:
https://github.com/ansible/ansible/blob/devel/test/integration/targets/ec2_elb_lb/tasks/main.yml#L41